.mergify.yml

```diff
@@ -1,28 +1,41 @@
 queue_rules:
     - name: default
       conditions:
-        - status-success="Validate"
-        - status-success=license/cla
-        - status-success=lint_test_build
+          - status-success="Validate"
+          - status-success=license/cla
+          - status-success=lint_test_build
 
 pull_request_rules:
-  - name: Automatic strict merge
-    conditions:
-      - base=master
-      - "#approved-reviews-by>=2"
-      - "#changes-requested-reviews-by=0"
-      - "#review-requested=0"
-      - status-success="Validate"
-      - status-success=license/cla
-      - status-success=lint_test_build
-      - label=ready-to-merge
-      - label!=do-not-merge
-      - title~=^(build|ci|chore|docs|feat|fix|perf|refactor|revert|style|test)(\([^)]+\))?:\s.+$
-    actions:
-      queue:
-        method: squash
-        name: default
-  - name: Automatic head deletion
-    conditions: []
-    actions:
-      delete_head_branch: {}
+    - name: Automatic boxmoji merge
+      conditions:
+          - author=boxmoji
+          - base=master
+          - files~=^[a-z]{2}-[A-Z]{2}\.properties$
```

Comment:
Is the matcher supposed to match the full file path, including directory names (i.e. files should start with `i18n/`)? or is it just the filename?

Reply:
no examples given 
![image](https://github.com/box/box-ui-elements/assets/103291617/fdafd641-7c24-401b-9aba-6c06397dfe9c)